app/controllers/admin_controller.rb

```diff
@@ -7,6 +7,37 @@ def summary; end
 
   def edit_metadata; end
 
+  def confirm_metadata
+    @params = params.permit(
+      :name,
+      :base_path,
+      :description,
+      :summary,
+      :document_noun,
+      organisations: [],
+      related: [],
+    )
+
+    @params[:organisations].reject!(&:empty?)
+    @params[:related].reject!(&:empty?)
+
+    @proposed_schema = @current_format.finder_schema.schema.merge(@params.to_unsafe_h)
+
+    @proposed_schema["signup_copy"] = "You'll get an email each time a #{@params[:document_noun]} is updated or a new #{@params[:document_noun]} is published."
```

Comment:
Is this old, and to be done as part of the email alerts card?

Reply:
Since signup copy is is consistently to the `document_noun` field I felt like it should be updated here at the same time.

An alternative would be that we remove the field entirely, what do you think?

Reply:
Ooh interesting! Feels like we either:

1) keep it here, or
2) remove the field configurability altogether (automatically construct it instead), or
3) move it to the new 'email' form and duplicate the document noun inline (i.e. "You'll get an email each time a foo is updated or a new foo is published."

I think I'm leaning toward 2. I can imagine some publishers might accidentally abuse the signup copy otherwise?

Reply:
Agreed, I've gone through and the only finder that doesn't quite match the proposed format is: `You’ll get an email each time a new UK Approved Recommendation is published.` and that's close enough I think.